tests/std/tests/P0718R2_atomic_smart_ptrs/test.cpp

```diff
@@ -222,6 +222,11 @@ void ensure_nonmember_calls_compile() {
     }
 }
 
+#ifndef __EDG__ // TRANSITION, DevCom-1656924
+// LWG-3661: constinit atomic<shared_ptr<T>> a(nullptr); should work
+constinit atomic<shared_ptr<bool>> a(nullptr);

```

Comment:
Should we also add coverage for the default constructed case?
```suggestion
constinit atomic<shared_ptr<bool>> a{};
constinit atomic<shared_ptr<bool>> b{nullptr};
```

Reply:
I'm building up a PR of tiny cleanups, I can look into this one later. I suspect we didn't notice it because it wasn't commented as TRANSITION, Clang 10.

Reply:
Thanks